supabase/migrations/20260504000001_seed_080_agk.sql

```
@@ -0,0 +1,73 @@
+  -- 081-09
+  ('081-09', '081-09-01', 'General',                                                            1),
+  ('081-09', '081-09-02', 'Fuel',                                                               2),
+  ('081-09', '081-09-03', 'Carburettor or injection system',                                    3),
+  ('081-09', '081-09-04', 'Air cooling systems',                                                4),
+  ('081-09', '081-09-05', 'Lubrication systems',                                                5),
+  ('081-09', '081-09-06', 'Ignition circuits',                                                  6),
+  ('081-09', '081-09-07', 'Mixture',                                                            7),
+  ('081-09', '081-09-08', 'Propellers',                                                         8),
+  ('081-09', '081-09-09', 'Performance and engine handling',                                    9)
+) AS s(topic_code, code, name, sort_order) ON s.topic_code = t.code
+WHERE t.subject_id = (SELECT id FROM easa_subjects WHERE code = '080')
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)